### ISSUE TITLE
issue with text-wrap?

### ISSUE THREAD

**Issue by jacobzlogar** [ChrisPenner/LumberJack#2]
Issue with text-wrap displaying the wrong portion of my logs as you can see from the provided screenshots both times i'm at the bottom of the file but when i toggle text-wrap with `w` the bottom of the file aka the most recent log stops updating after a certain point
<img width="644" alt="screen shot 2017-01-23 at 5 03 40 pm" src="https://cloud.githubusercontent.com/assets/10762368/22224796/ee17acd2-e18d-11e6-8a88-16740e09dbff.png">
<img width="642" alt="screen shot 2017-01-23 at 5 03 29 pm" src="https://cloud.githubusercontent.com/assets/10762368/22224797/ee1cfa52-e18d-11e6-8754-c00bb826646c.png">

**Comment by ChrisPenner** [comment]
Thanks for the issue; This is caused by the number of lines displayed to screen not matching the `actual` number of lines; it may be a bit tricky to fix since the scrolling code is decoupled from the rendering code (which is in a library). Perhaps there's a better utility for this now than when I originally wrote it.

I don't have a ton of time to spend on this at the moment; but if you want to take a look at it I can definitely help point you towards the right places!

**Comment by jacobzlogar** [comment]
Okay, i was just wondering if this was user error or something else, thanks for the response!

**Comment by ChrisPenner** [comment]
Nope; not your fault haha! I'll try to take a look at this when I can; but no promises that it'll be soon haha! I'll leave this open as a reminder for later.
